consul_config.py.ctmpl

```diff
@@ -18,6 +18,8 @@ SQLALCHEMY_DATABASE_URI = "postgresql://listenbrainz:listenbrainz@{{.Address}}:{
 MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = "postgresql://messybrainz:messybrainz@{{.Address}}:{{.Port}}/messybrainz"
 POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/postgres"
 {{end}}
+{{else}}
+KEYDOESNOTEXIST = "pgbouncer-master is down"
```

Comment:
I'd prefer that this was something like
`SQLALCHEMY_DATABASE_URI = "SERVICEDOESNTEXIST_pgbouncer_master"`
or similar. Add it for all config items inside the `if`. This is the same motivation as adding the key name to the KEYDOESNTEXIST message.

By adding a field for each config item, we actually allow the service to load a config if we want to.
Currently when apps start up, if a service is missing they try and read a config item but fail. I think it's better to be able to read the value, but then fail at the next stage.
I'm happy to discuss this, as I'm not 100% convinced that it's the right thing to do.

Reply:
Makes sense to me. One possible issue could be whether the service is able to handle an incorrect config. For API keys I think this could be an issue. But for database connections I think the connection would just fail logging an exception which is fine.

Reply:
yes, that's a valid concern. The hope here is that we'll get a notification before the service starts anyway, so this might not be as much of a problem